client/src/app/components/Layout/Layout.jsx

```diff
@@ -53,13 +53,13 @@ export class Layout extends Component {
       <Fragment>
         <Notifications />
         <Switch>
+          <Route exact path="/" render={() => <Redirect to="/dashboard" />} />
```

Comment:
why such a change??

Reply:
The warning appeared when the url was typed manually in a browser or after a page refresh, the router always went back to the root and always tried to load the Dashboard first and run some setStates statements which caused a warning after loading proper component:  Can’t call setState (or forceUpdate) on an unmounted component...